modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -100,6 +100,34 @@ Status description:: error: connection exception - no such routing policy. Polic
 Database exceptions occur when a client request contains the wrong format, types, or other unsupported input.
 Some examples are data and constraint creation, which conflicts with existing constraints, properties of non-storable type, and spatial and temporal values with invalid components.
 
+=== 22000
+
+Status description:: error: data exception
+
+=== 22001
+
+Status description:: error: data exception - string data, right truncation
+
```

Comment:
This one doesn't seem to be used after all. Maybe I mixed it up with 22N01 when updating the spreadsheet. SO you can comment it out for now.